pkgs/development/python-modules/testpath/default.nix

```diff
@@ -17,6 +18,11 @@ buildPythonPackage rec {
     pytestCheckHook
   ];
 
+  preCheck = lib.optionalString stdenv.isDarwin ''
+      # Work around https://github.com/jupyter/testpath/issues/24
+      export TMPDIR="/tmp"
```

Comment:
```suggestion
    # Work around https://github.com/jupyter/testpath/issues/24
    export TMPDIR="/tmp"
```

Reply:
Yeah, this came up last time as well: https://github.com/NixOS/nixpkgs/pull/126642#discussion_r650556767